README.md

```diff
@@ -1 +1,88 @@
-# Books-store
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Book Store
+
+> This is an Educational project aimed at learning professional use of React/React-Redux. It features books management; Adding, Editing, Completing and Removing books from the system using Redux actions and reducers. This app is MIT Licensed
+
+## Built With
+
+- JavaScript
+- ReactJs,
+- Git, GitHub, Git workflow
+
+## Live Demo
+
+[Link]()
+
+## Getting Started
+
+### To have a copy of this project running on your local machine, follow these commands:
+
+- Clone this repo to your local machine
+  - `https://github.com/AshwinCarvalho999/Books-store.git`
+  - Open the folder "Books-store and run `npm install` to install all necesary scripts,
+  - Run from your termminal, `npm start` to open the project in your browser.
+
+**This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- [NodeJS](https://nodejs.org/en/docs/)
+
+> As an asynchronous event-driven JavaScript runtime, Node.js is designed to build scalable network applications. In the following "hello world" example, many connections can be handled concurrently. Upon each connection, the callback is fired, but if there is no work to be done, Node.js will sleep.
+
+- [Text Editor(VScode Recommended)](https://code.visualstudio.com/)
+
+> Visual Studio Code is a free coding editor that helps you start coding quickly. Use it to code in any programming language, without switching editors.
+
+### Setup
+
+**Run from your terminal:**
+
+- `npx create-react-app todo-list-react`
+
+**cd into Books-store folder, then run from your terminal:**
+
+- `npm start`
+
+### Usage
+
+\*\*When the above commands aresuccessfully run, your browser should open on `localhost:3000`, if not, open your browser and load `127.0.0.1:3000` to see your default template loaded
+
+### Run tests
+
+> coming soon :smile:
+
+### Deployment
+
+> coming soon :smile:
+
+## Author
+
+👤 **Ashwin Carvalho**
+
+GitHub: [GitHub](https://github.com/AshwinCarvalho999)
+LinkedIn: [LinkedIn](https://www.linkedin.com/in/ashwin-carvalho-5426701b1/)
+Twitter: [twitter](https://twitter.com)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] The MIT link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯